src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/ClaimIssueForSpecCallbackHandlerTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.caseevents;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.handler.callback.BaseCallbackHandlerTest;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.sampledata.CaseDataBuilder;
+import uk.gov.hmcts.reform.civil.service.DeadlinesCalculator;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static java.time.LocalDate.now;
+import static java.time.LocalTime.MIDNIGHT;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {
+    ClaimIssueForSpecCallbackHandler.class,
+    JacksonAutoConfiguration.class
+})
+class ClaimIssueForSpecCallbackHandlerTest extends BaseCallbackHandlerTest {
+
+    @Autowired
+    private ClaimIssueForSpecCallbackHandler handler;
+
+    @Autowired
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @MockBean
+    private DeadlinesCalculator deadlinesCalculator;
+
+    private final LocalDateTime deadline = now().atTime(MIDNIGHT);
+
+    @Test
+    void shouldAddClaimNotificationDeadline_whenClaimIsIssued() {
```

Comment:
For all tests:

add `// Given`, `// When`, `// Then` comments before each given/when/then section of the tests. It improves their readability.

Reply:
Done